[DO NOT MERGE]Add TDEBYOK api swagger files for serveroperations and MIoperations

Yueren-Wang opened the pull request:
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

yuerenwa committed:
addtdebyokapiswaggers

yuerenwa committed:
addtdebyokapiswaggers

msftclas commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/not_signed)](https://cla.opensource.microsoft.com/Azure/azure-rest-api-specs?pullRequest=7551) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/Azure/azure-rest-api-specs?pullRequest=7551) before we can accept your contribution.<br/><br/>:x: kingsleyAzure [sign now](https://cla.opensource.microsoft.com/Azure/azure-rest-api-specs?pullRequest=7551)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/Azure/azure-rest-api-specs?pullRequest=7551) it.</sub>

azuresdkci commented:
Can one of the admins verify this patch?

AutorestCI commented:
# Automation for azure-sdk-for-python

AutorestCI commented:
# Automation for azure-sdk-for-go

kingsleyAzure commented:
Please take a look, Thanks !

Commit:
Fix typo in readme.go for web (#7547)

hyonholee committed:
Compute 2019-07-01 version (#7094)

* Copy compute.json and runCommands.json from 2019-03-01 to 2019-07-01

* changes to add publicIpAddressVersion field (#7173)

* temp fix

* Adding new VirtualMachines.POST.Reapply action to compute api-version: 2019-07-01. Updating parameter info for skus api-version:2019-04-01..  (#7167)

* Copy compute.json and runCommands.json from 2019-03-01 to 2019-07-01

* Adding new VirtualMachines.POST.Reapply action.

* Adding example for POST Reapply virtual machine action.

* Adding missing $filter parameter documentation to skus.json for api-version: 2019-04-01.

* Unexpected change.

* Added DiskEnryptionSet property in the managed disk parameters for the VM/VMSS (#7293)

*  updated swagger

*  updated swagger

* Removed example files which are not relevant and added links to examples for newly added property

* fixed the syntax error

*  updated swagger

*  fixed the sytax error

* Added examples for the DiskEncryptionSet property (#7318)

*  updated swagger

*  updated swagger

* Removed example files which are not relevant and added links to examples for newly added property

* fixed the syntax error

*  updated swagger

*  fixed the sytax error

*  updated code

*  udpated swagger to create DiskEncryptionSet as new property to initiate

* Resolved conflicts with DiskRP definition names for the DiskEncryptionSet (#7340)

*  updated swagger

*  updated swagger

* Removed example files which are not relevant and added links to examples for newly added property

* fixed the syntax error

*  updated swagger

*  fixed the sytax error

*  updated code

*  udpated swagger to create DiskEncryptionSet as new property to initiate

* resolved conflicts and updated swagger due to conflict in the DiskRP definitions

* Disk 2019-07-01 version (#7280)

* Copy compute.json and runCommands.json from 2019-03-01 to 2019-07-01

* changes to add publicIpAddressVersion field (#7173)

* Add diskEncryptionSet in swagger compute-2019-07

* resolve semantic conflicts

* Fix model conflicts

* Resolve readme

* Resolve readme

* Resolve description conflicts

* Improve description

* Fix spell error

* Add some examples.

* fix model error

* Update examples (#7429)

* Copy compute.json and runCommands.json from 2019-03-01 to 2019-07-01

* changes to add publicIpAddressVersion field (#7173)

* Add diskEncryptionSet in swagger compute-2019-07

* resolve semantic conflicts

* Fix model conflicts

* Resolve readme

* Resolve readme

* Resolve description conflicts

* Improve description

* Fix spell error

* Add some examples.

* fix model error

* Update examples

* Add extension API for VMScaleset VMs (#7407)

* add extension Api

* change operationID

* update description

* update description

* add examples

* add missing /

* add missing / in examples

* add new line

* remove unnecessary file

* add reference examples

* fix model validation failures

* remove provisioning state from params

* remove unnecessary param

* fix Delete example

* updating the description of storage account id (#7385)

* Private temp (#7456)

* Copy compute.json and runCommands.json from 2019-03-01 to 2019-07-01

* changes to add publicIpAddressVersion field (#7173)

* Add diskEncryptionSet in swagger compute-2019-07

* resolve semantic conflicts

* Fix model conflicts

* Resolve readme

* Resolve readme

* Resolve description conflicts

* Improve description

* Fix spell error

* Add some examples.

* fix model error

* Update examples

* Remove uniqueId in swagger

* Remove required for diskEncryptionSetId

* cherry pick code owner change to compute-2019-07 (#7480)

* Iops, throughput and instance view (#7499)

* Swagger changes for IOPS, Bulk InstanceView, documentation for GETVM and InstanceView

* Fixed typos

* Updated samples

* Update 2019-07 from latest 2019-03 update.

* Update swagger (#7510)

* Copy compute.json and runCommands.json from 2019-03-01 to 2019-07-01

* changes to add publicIpAddressVersion field (#7173)

* Add diskEncryptionSet in swagger compute-2019-07

* resolve semantic conflicts

* Fix model conflicts

* Resolve readme

* Resolve readme

* Resolve description conflicts

* Improve description

* Fix spell error

* Add some examples.

* fix model error

* Update examples

* Remove uniqueId in swagger

* Remove required for diskEncryptionSetId

* Update principalId and tenantId to be readonly

* small space fix

* add update property (#7514)

* Sync with the latest 2019-03-01 version to 2019-07-01 version of compute.

* suppress LintDiff validation error.

* Add default responses to new APIs.

Commit:
regenerated all-api-versions

asarkar84 committed:
Correcting the verbs from Create BillingProfile and InvoiceSection as per code implementation (#7524)

* Billing RP spec changes for listInvoiceSectionsWithCreateSubscriptionPermission API

* Updated the operation id

* Added x-ms-pageable

* Added next link

* Deleted unused file ProductCancelByBillingAccount

* Incorporated review comments

Incorporated review comments

* Added new properties in Invoicing APIs

Added new properties in Invoicing APIs

* Incorporated review comments

Incorporated review comments

* Added invoiceEmailOptIn in the UpdateBillingProfile request

Added invoiceEmailOptIn in the UpdateBillingProfile request

* Update billing.json

* Updating the verbs from Create BillingProfile and InvoiceSection entities

Updating the verbs from Create BillingProfile and InvoiceSection entities

sornaks committed:
Updating swagger spec to fix billing plans schema in catalogs response. (#7541)

romahamu committed:
Microsoft Azure StorageCache new API version (#7520)

* Update autorest config and swagger to support new API version 2019-11-01.

* Re-write description of some of the responses in Cache_Create.json and storagecache.json.

* Description updates.

Ready for re-review.

* Remove references to PATCH storagetarget

* Update C/ST_Create to C/ST_CreateOrUpdate

* Fix formatting.

* Fix tag name to align with previous version tags and GO SDK namespace.

ruowan committed:
update ARM email template and fix repoWhiltelist location (#7556)

andyzhangx committed:
add AKS rotateClusterCertificates API (#7436)

* add aks rotate-certs operation for 2019-08-01

* feat: use 204 instead of 200

Commit:
[Storage] Sync swagger changes from 2019-04-01 to 2019-06-01 apiversion (#7552)

* [Storage] Sync swagger changes from 2019-04-01 to 2019-04-01 apiversion

* Removing skiptoken which is not being used by the service

* The skiptoken is not used by the service, removing it from latest swagger.

omar-el-hariry committed:
Omelhari/feature assigning ap is (#7503)

* Adding feature assigning APIs

* Remove Pattern Feature APIs

* fix build

yangyuan committed:
Refactory CognitiveServicesAccount model specification. (#7415)

* Refactory CognitiveServicesAccount model specification.

* fix samples

zfchen95 committed:
[SRP] Add CORS to common types (#7562)

rokulka committed:
Updating the docs to include rankerType field in QueryDTO (#7538)

* Updating the docs to include rankerType field in QueryDTO

* PR comments

bleroy committed:
[Azure Search] Apply branding changes to stable/2019-05-06 and track1/stable/2019-05-06 (#7511)

* Apply branding changes to stable/2019-05-06 and track1/stable/2019-05-06

* Address code review comments

farehar committed:
Add Hybrid RP rest api spec (#7487)

* Hybrid RP swagger

* Addressed feedback

* fix typo

* fix build issues

* fix validation issues

* fix validation errors

* fix errors

* fix errors

* fix errors

* remove 204 return codes

* updated examplese

Commit:
regenerated all-api-versions

anton-evseev committed:
Merge Network's October release branch into master (#7564)

Commit:
regenerated all-api-versions

wanha-ms committed:
update-event-schema-for-azure-storage-event-grid-notification-for-cr… (#7509)

filizt committed:
[Resources] Add Deployment scripts as a new resource type - preview - (#7486)

* Approved deployments scripts swagger from private repo

* Update readme files and fix typo

* Update property letter casing

* Removing the containerSettings property due to network profile not supporting user assigned MSI.

* Update specification/resources/resource-manager/readme.csharp.md

update package name

Co-Authored-By: Yeming Liu <felix_liu@outlook.com>

Commit:
regenerated all-api-versions

bganapa committed:
Fixing issues in Azurestack admin api specs while running the powershell generator (#7566)

Commit:
fixing tag package-2019-07-01-only (#7574)

Commit:
Publish container service 2019-10-01 api version (#7567)

* Adds base for updating Microsoft.ContainerService from version stable/2019-08-01 to version 2019-10-01

* Updates readme

* Updates API version in new specs and examples

* added private FQDN (#7430)

* added private FQDN

* added example

* remove example

* updating sdk tags for 2019-10-01 api version (#7550)

* updating sdk tags for 2019-10-01 api version

* fixing ci errors

Commit:
updates for network experiment (#7573)

Commit:
scope assignmens override + flatten all (#7571)

Commit:
regenerated all-api-versions

ruowan committed:
remove 5days email notification (#7576)

Commit:
updating peering cli structure (#7572)

jaredmoo reviewed (approved):
Looks good. If/when the APIs are available in at least 1 production region, please add the files to readme.md so that they can be included in generated SDKs.

ArcturusZhang committed:
Add 2019-06-01 in storage for golang (#7578)

kalyanaj committed:
EventGrid: New preview API version with support for additional destination types, CloudEvents 1.0, and batching support. (#7527)

farehar committed:
Hybrid RP. enable delete operation (#7568)

* Hybrid RP swagger

* Addressed feedback

* fix typo

* fix build issues

* fix validation issues

* fix validation errors

* fix errors

* fix errors

* fix errors

* remove 204 return codes

* updated examplese

* enable delete operation

* Hybrid RP swagger

* Addressed feedback

* fix errors

* fix errors

* fix errors

* updated examplese

* enable delete operation

* fix tyop

* fix spaces

Commit:
test generation for web (#7587)

* test generation for web

* missing things

ArcturusZhang committed:
Fix error in luis runtime readme (#7593)

ArcturusZhang committed:
Change go sdk from tag 2019-07-01 to 2019-07 (#7592)

Commit:
some naming replacing in examples to make it inline with cli commands (#7595)

* some naming replacing in examples to make it inline with cli commands

* undo unnecessary change

* one more missing

* more unnecessary changes undone

* undo

* undo

Commit:
Network experiment structure update (#7599)

* updated next experiment client structure

* update examples to match azure cli standards

LingyunSu committed:
Fix miss spelling property name (#7586)

Fix miss spelling property name externalDnsIPAddress

ramnov committed:
Swagger update for Azure Private Link Integration for Azure Database for PostgreSQL, MySQL and MariaDB (#7585)

Commit:
regenerated all-api-versions

ajay-sainy committed:
[Hub Generated] Review request for Microsoft.ManagedServices to add version stable/2019-09-01 (#7426)

* Adds base for updating Microsoft.ManagedServices from version stable/2019-06-01 to version 2019-09-01

* Updates readme

* Updates API version in new specs and examples

* Using the filter for collection call to return specific definitions.
Created separate path for point get call.

Previous comment: New api-version, having new resource type - marketplaceregistrationdefinition.

Commit:
regenerated all-api-versions

fore5fire committed:
[CDN] Add missing customHttpsParameters field of CustomDomainProperties. (#7441)

* CDN: Add missing customHttpsParameters field of CustomDomainProperties.

* Add minimumTLSVersion field to CustomDomainHttpsParameters in latest CDN API

Commit:
added missing part from 2019-08-01 (#7614)

ArcturusZhang committed:
[READY TO MERGE] Add new composite version for sql (#7590)

* Add new composite version for sql

* Changed to v3

anton-evseev committed:
Use node.js 10.16.3 in Network validation (#7607)

fengzhou-msft committed:
add test for peering (#7618)

Commit:
Update PULL_REQUEST_TEMPLATE.md (#7619)

Fix #7522

laurawu19 committed:
Update ResourceIdentity to EncryptionSetIdentity (#7617)

Commit:
adding swagger-to-sdk section (#7601)

xseeseesee committed:
Storage: add package-2019-06 in readme.java.md (#7575)

NNara committed:
Changed WebActivity and webhook activity auth to support both AKV and Secrets (#7554)

Commit:
Update data set related documentation comments (#7582)

bowen5 committed:
1. Make PUT and PATCH against App resource supports async. (#7594)

2. Remove "Processing" enum from DeploymentResourceProperties's status property.
3. Rename "Processing" enum to "Updating" for DeploymentResourceProperties's provisioningState.

mozehgir committed:
Create SwaggerValidationTools.md

mozehgir committed:
Update SwaggerValidationTools.md

mozehgir committed:
Create Getting started with OpenAPI specifications.md

arv100kri committed:
Rename 'Azure search index' -> 'index' (#7628)

j-so committed:
Add new EventGrid event type for Azure Machine Learning Services (#7608)

Commit:
regenerated all-api-versions

Jingshu923 committed:
Update Db2LinkedServiceTypeProperties (#7546)

stack111 committed:
adding missing readonly property (#7428) (#7606)

shleiAmy committed:
add JSONWebKeySet in attestation.json file (#7490)

yungshinlintw committed:
Layout API update in FormRecognzier 2.0-preview (#7581)

* Adding FR v2.0 API updates

* PR Validation: Fix typo, missing file ref.

* PR Validation: Fix missed typo

* PR Validation: fixing model verification errors, doc string fixes.

* Fixing model verification, added Receipts endpoints, related examples.

* Fixing semantic validation error

* Update labelFile param

* fixing types

* Fixing model validation errors

* Fixing model verification errors, other fixes.

* Fixing mode verification error.

* Fixing typo.

* Fixing doc strings, Location header update, Change response code for Analyze calls

* Incorporating feedback from Paul.

* Feedback from Florian, verification fix.

* Fixing for merge verification.

* fixing for model verification tests

* fixing model verification errors.

* fixing example file to correct useLabelFile option.

* Update for Layout Analysis

* Update result for layout

* Update for OCRLanguage parameter

* Update from PR comment

* Remove language as a parameter

farehar committed:
Disable api generation for operations not supported in public preview (#7625)

* Hybrid RP swagger

* Addressed feedback

* fix typo

* fix build issues

* fix validation issues

* fix validation errors

* fix errors

* fix errors

* fix errors

* remove 204 return codes

* updated examplese

* enable delete operation

* Hybrid RP swagger

* Addressed feedback

* fix errors

* fix errors

* fix errors

* updated examplese

* enable delete operation

* fix tyop

* fix spaces

* disable api generation for operations not supported for public preview

addesai committed:
Remove resource group from allowed scopes for usageDetails (#7630)

Changes to remove resource group from the list of allowed scopes for UCDD given it's not supported.

jimmyca15 committed:
Fixed misnamed sku in examples. (#7623)

addesai committed:
Fix documentation (#7632)

fengzhou-msft committed:
add tag 2019-06 to python readme (#7634)

ruowan committed:
update send email to assignee config (#7636)

arv100kri committed:
[Azure Search] Introduce 2019-05-06-Preview swagger (#7615)

* Add Microsoft.Azure.Search.Data 2019-05-06-Preview to track1 and 'track2'

* Add Microsoft.Azure.Search.Service 2019-05-06-Preview to track1 and 'track2'

* Add codegen tags

* Fix 'track2' directory casing

* Fix 'track1' directory casing

* Fix API versions

* Add encryption key to swagger

* Update branding changes

Commit:
regenerated all-api-versions

mozehgir committed:
Create Breaking changes guidelines.md

mozehgir committed:
Update Getting started with OpenAPI specifications.md

mozehgir committed:
Update README.md

Commit:
Viviali iotcentral new API onboarding (#7570)

* Adding new API surface

* With examples

* addressing validation errors1

* Updated examples

* Fixes to IoTCentral Swagger and Examples - Example Validation Passing

* added cde changes

* Adding readme.md autorest section and continuos data export

* adding iotcentral customwords

* iotcentral custom words

* Add Data Planse Parent Folder Structure

* Fix readme.md input files pathing

* Updating API token create

* update readme for avocado

* readme.md remove unused example files

* remove uncessary files

* Changed instance to component

* updated swagger

* Fixed swagger

* Adding getapplication/listpplication calls

* remove conflicting host

* Testout parameter Change

* inline common parameters to methods to pass pipeline tooling swagger parsing bug

Commit:
regenerated all-api-versions

tfitzmac committed:
use secure link for documentation

romahamu committed:
Microsoft Azure StorageCache - Add 204 response code to cache flush, start, stop and upgrade operation. (#7584)

olterner committed:
Add support for None for MSI RP (#7600)

* Add support for None for MSI RP

* Fix comment

mpoliako committed:
Update Microsoft.VMwareCloudSimple provider schema to 44 release of Cloudsimple (#7497)

nachakra committed:
Multi-origin support for CDN (#7604)

* Publish 2019-04-15 API version to make reviewing new API changes easier

* Changes for new API version 2019-12-31

* Update readme.md

* Correct spellCheck errors

Commit:
regenerated all-api-versions

swhite-msft committed:
Adding EU compliance not to description (#7610)

LingyunSu committed:
Fix miss spelling property name and case sensitive (#7616)

ArcturusZhang committed:
Add 2019-10-01 (#7657)

ArcturusZhang committed:
Remove api-version for composite-v3 (#7658)

Commit:
Attestation integration tests and cli module (#7648)

* updated readme

* fixed too many replacements

* additional example update

rafarhat committed:
iot central minor swagger changes and description fixes (#7656)

* minor swagger changes and description fixes

* linter fix trial

* linter fix trial 2

* Capability Definition Linter Final Fix

jihochang committed:
Remove ReplicaSet Nested Resource (#7627)

* Remove ReplicaSet Nested Resource

* Fix readyonly properties

ChenTanyi committed:
App Configuration: add java readme (#7659)

Commit:
Add stable version for Data Share and in place sharing support for Kusto (#7641)

* Update data set related documentation comments

* Add inplace data sharing support for Kusto

* Add shell stable version

* Update stable version for easy diff

* fix readme files

* update path for readme.go.md

Commit:
regenerated all-api-versions

Commit:
Update from latest master (#7638)

Commit:
Add Prettier (#7637)

ayanMSFT committed:
Cosmos DB Private Endpoint Connection Swagger  (#7563)

* Added Private Endpoint Cosmos DB Swagger spec

* Deleted generated

* Fixing ErrorResponse reference

* PrivateEndpointConnections_CreateOrUpdate has no 200 return, and removing 200 from CosmosDBPrivateEndpointConnectionDelete.json

* Moved Defeinitions to common-types/types.json, and removed references to stable paramaters/definitions

* Fixing Update example

* Fixing Update example

* Moving defs back into privateEndpointConnection.json

* reverting changes to types.json

Commit:
regenerated all-api-versions

asinn826 committed:
Update serial console swagger (#7444)

* Update serial console swagger 10/9/2019

* Make operationId casing consistent

* Fix formatting. Also, default is not a parameter

* Address build errors

* Describe GetOperations output

* Clarify scale set instance

* Add default back as a parameter, update examples accordingly

* Add x-ms-parameter-location

* rename GetSerialConsoleDisabledResult to SerialConsoleStatus and update example accordingly

emgu-ms committed:
Failover Database Readable Secondary (#7405)

* Add swagger changes for failover databases and failover elastic pools

* Fixed api version for FailoverDatabases

* Add examples

* Add to read me

* Add swagger files for failover database readable secondaries

* Remove FailoverDatabaseReadableSecondary.json and include the extra parameter in FailoverDatabase.json instead

yeskarthik committed:
[TimeSeriesInsights] Query categorical variable and warm store query parameter changes (#7609)

* Fix or suppress spec violations
Fix R2016 by making PATCH body properties optional. Suppress the remaining 4 violations that are confirmed as false positives.

* Try removing 2nd top-level heading, which _may_ have been causing a linter warning.

* Fix extra indention which prevented linter from running.

* Added dataStringComparisonBehavior parameter to reference data put/update spec.

* Fixed grammar.

* adding partitionkeyProperty to environmentInfo

* changing property name

* changing modelAsString

* Initial checkin of spec. Unchanged from GA release other than version.

* Remove .orig files

* Fix validation error by removing IoTHub ES from List ES example response

* Add extra example that was missing from preview spec.

* Add long-term environment to resource hierarchy, and update examples.

* Switch to model-as-string for enums. Restore EnvironmentResource's inheritance from TrackedResource.

* Upsate sku description

* Add new TSI preview api-version to readme.md.

* Add suppressions for false-positive linter failures.

* Test suppression failures

* Adding preview package tag to several other places in readme.md

* Add preview package to the readme.go.md file

* Remove invalid required property from EnvironmentCreateOrUpdateParameters

* Initial version of data plane swagger spec for PAYG (Preview) SKU.

* Added missing Readme.md

* Fixed name of readme

* Fixed path to swagger json

* Attempt to fix readme

* Apply suggestions from code review

Co-Authored-By: dmdenmsft <dmden@microsoft.com>

* Update timeseriesinsights.json

* Update timeseriesinsights.json

* Added link to oav issue

* Added changes for categorical variable kind and warm store query parameter in queries.

* Fix interpolation casing.

* Fixed encoding, spelling errors and avocado errors.

* Added long variables to categorical examples.

* Added long type and updated examples.

mfrintu committed:
Web activity changes for IR (#7605)

wenjlu-lgy committed:
Change storage account output information. (#7591)

anton-evseev committed:
Improve Network's descriptions (#7662)

zfchen95 committed:
[SRP] Create separate example for enable AD on storage account (#7664)

shurd committed:
Cosmos DB patch and multi-api support (#7336)

* adding patch support in new API version, adding multiapi support for python, go, java, adding new property from database account resource, adding locations to response of database accounts

* removing api kind from one example

* adding partition key version, removing new property from old api version

* revert changes to old api swagger

* rebasing and adding preview versions to sdk generation

* create or update method with guidance in the description

* renaming create variables

* minor text changes

Commit:
Fix scripts/switch-to-preproduction (#7666)

* fix scripts/switch-to-preproduction

* Add tsc to postinstall

liubing-microsoft committed:
remove licenseKey from required (#7647)

Commit:
Add prettier fix docs and prettier support for vscode (#7667)

nebadr committed:
Update LUIS Authoring v3.0-preview swagger (#7661)

* update swagger

* Remove unused examples

* Revert "Remove unused examples"

This reverts commit 99e6f151e0e5944e5513cd601e49da82178e0438.

* Revert "update swagger"

This reverts commit 80df648593854cd3baa52059d3215a8dc42f4399.

* Change url

* Add missing field

* Adding import changes

* Changes

* Add hierarchicals

xseeseesee committed:
EventGrid: add package-2020-01-preview in readme.java.md (#7620)

lagalbra committed:
Update HanaOnAzure Swagger to support new Cascade Lake SKUs (#7670)

* Add Cascade Lake SKUs: S224oo, S224oom, S224ooo

Add other SKUs which had not been synced: S112

* Add examples with new SKUs

pilor committed:
Microsoft.Authorization/policy* entities 2019-09-01 (#7380)

* Copy 2019-06-01 to 2019-09-01

* Api-version 2019-09-01 of policy entities

* Allow multiple groups

* Explicitly model parameter definitions and values

Tiano2017 committed:
Make 'location' required for sub/mg/tenant deployment. (#7543)

bowen0106 committed:
Fix the typo for actionsRequired in PrivateLinkServiceConnectionState. (#7672)

kfbehar committed:
Add Azure Security Center for IoT to productFilter enum (#7645)

zfchen95 committed:
[SRP] Add delete retention policy for blob container and file share (#7626)

shawncx committed:
Add Properties for ssis embedded package (#7646)

mmyyrroonn committed:
fix ruby.md for release (#7680)

ramoka178 committed:
Managementgroups/20191101 (#7673)

* new api version for management groups

* modify type in MG entities

* moving to stable folder

* fix example

* update read me

* fix model validations

* PR comment resolved

* remove input file

* change file extension case

* Rename management.JSON to management.json

* fixed checks

Commit:
regenerated all-api-versions

sarangan12 committed:
Fix issue with CloudError in Storage Cache (#7684)

vrushg-ms committed:
[Ready-To-Merge]Dev mlservices 2019 11 01 - private repo pr Azure/azure-rest-api-specs-pr#902 (#7267)

juniwang committed:
fix missing status code in SignalR (#7635)

raghushantha committed:
Make OSName, OSVersion properties writable (#7674)

ArcturusZhang committed:
Fix wrong preview pacakge path (#7685)

bvanderlugt committed:
updating with warm store features (#7643)

sriramvu committed:
[Hub Generated] Review request for Microsoft.Subscription: Removed an unused API to make things simple, also corrected examples (#7540)

* Removed unused API to make things simple, also corrected examples

* Taking comments to be consistent within this API version

* Update createSubscription.json

Making minor change to rerun the checks as public.rest-api-specs check is reporting as running for more than a day when it has actually got completed.

hillaryc committed:
add azureActiveDirectoryTenantId (#7681)

* add azureActiveDirectoryTenantId

Add azureActiveDirectoryTenantId as recommended by the Azure Active Directory team.

* fix typo Diretory

kerberos02 committed:
Added new api version 2019-09-01, which changed the soft delete behavior (#7639)

* Copy files of api version 2018-02-14.

* Added new api version 2019-09-01, which changed the soft delete behaviour: new key vault will default to  soft-delete being on (or true), if not explicitly specified.

* Fixed CI errors: The example JSON file is not referenced from the swagger file.

* Fixed Model Validation errors.

* Fixed model validation errors(2nd time).

* Fixed validation error.

Commit:
add list of tests to be generated for internet-analyzer (#7693)

LijuanZ committed:
[Datafactory] Add reportStatusOnCallBack property to webhook activity (#7682)

* Add reportStatusOnCallBack property to webhook activity

* Change type to object

Commit:
Fixed scope assignments and some documentation (#7665)

ArcturusZhang committed:
Fix output-folder error in azuredata (#7695)

yaohaizh commented:
@kingsleyAzure are you still follow this PR? Could you help fix the CI errors?

ArcturusZhang committed:
Fix namespace error in azuredata for golang (#7696)

vihnguyen committed:
[Hub Generated] Review request for Microsoft.Consumption to add version stable/2019-10-01 (#7653)

* Added billingFrequency docs for ARM RI transaction

* Added billingFrequency docs for ARM RI transaction + Remove unsupported resourceGroup scope for Marketplace

filizt committed:
Add new provisioning state (#7687)

HimanshuChhabra committed:
[Routing Preference] Adding routing preferences API changes, private repo PR# 973 (#7689)

* [Routing Preference] Adding routing preferences API changes, private repo PR#973

* Addressing Spell check validation, changed the description

avidhanju committed:
[Hub Generated] Review request for Microsoft.Insights to add version preview/2019-10-17-preview (#7611)

* Adds base for updating Microsoft.Insights from version preview/2018-06-17-preview to version 2019-10-17-preview

* Updates readme

* Updates API version in new specs and examples

* Adding initial files for Workbook templates.

* Fixing required fields.

* Adding PATCH.

* Fixing example ref for patch.

* Fixing references.

* Adding description for a property.

* Fix typo.

* Fix required properties.

* Fixing parameter types.

* Remove TagsParams.

* Fix another data type.

* Fix read only properties.

* More fixes.

* Renaming resource parameter.

* Removing operations file.

* Adding operations file.

* Fixes typo.

* Fix name.

* Adding workbooktemplates to custom words.

* Removing the operations api.

* Fix readme.

* Fixes from code review.

* Fixing examples schema.

* Fix Delete example response code.

* Fix default package in readme.

Commit:
regenerated all-api-versions

qiaozha committed:
add clear-output-folder in resource-manager readme.typescript.md to fix js sdk build failure (#7676)

shsingla committed:
Swagger changes for Microsoft.Migrate namespace  (#7063)

junyezhu committed:
add PreferredIPVersion to ConnectivityCheck (#7205)

ruowan committed:
remove waitForAzureAPIBoard label for SLA-bot (#7707)

Commit:
Fix prettier-check doc (#7697)

zfchen95 committed:
[SRP][Fix Typo] DeleteRetentionPolicy is a common class that applies to both file and blob (#7703)

Commit:
upgraded integratino test for healthcare apis (#7708)

eladperets committed:
Add Microsoft.PolicyInsights/policyMetadata resource type spec (#7675)

zfchen95 committed:
[SRP] Remove a recent added property for container soft delete (#7706)

Commit:
regenerated all-api-versions

HimanshuChhabra committed:
[Routing Preference] : Updated the Flatten property and added example . Linked to PR#7689 (#7714)

* Updating flatten property to false

* Updating examples

fengzhou-msft committed:
add cli readme for appconfiguration (#7710)

davidzhaoyue committed:
[Datafactory] Enable SQL DW Copy Command (#7700)

* [Datafactory] Enable SQL DW Copy Command

* Add MAXERRORS into custom-words.txt

punit-bhatt committed:
Adding swagger changes for GET and LIST replicationEligibilityResults API [Do Not MERGE] (#7698)

viananth committed:
Azure stack deployment resource provider admin open api specification (#7561)

Commit:
regenerated all-api-versions

ruowan committed:
fix SLA-bot filter expression (#7719)

liranc committed:
Add 'unmaskedIpLoggingStatus' to IotSecuritySolution (#7283)

* Added unmaskedIpLoggingStatus field

* Converted tabs to spaces

Khushboo-Baheti committed:
Fix filterdata example for VirtualNetworkGateway and VirtualNetworkGatewayConnection (#7718)

* FilterDataChange

* prettier

kingsleyAzure commented:
I am still working on it

yuerenwa committed:
add inaccessible database and inaccessible managed database swaggers and examples

yuerenwa committed:
add inaccessible database and inaccessible managed database swaggers and examples

tfitzmac committed:
remove locale from links (#7722)

zfchen95 committed:
[SRP] Add account sku to file/blob service response (#7694)

* [SRP] Add account sku to file/blob service response

* Fix the sku type

* Make skuName and tier common attributes

yuerenwa committed:
fix ci error

yuerenwa committed:
fix ci error

yuerenwa committed:
add all files

yuerenwa committed:
add all files

yuerenwa committed:
fix MA

yuerenwa committed:
fix MA

yuerenwa committed:
1

yuerenwa committed:
1

yuerenwa committed:
fix some example error

yuerenwa committed:
fix some example error

tfitzmac committed:
remove locale from link (#7723)

qiaozha committed:
upgrade rest-api-specs-scripts version (#7712)

Nilambari committed:
Fix Virtual Wan Point to site feature resources property names from swagger. (#7498)

yuerenwa committed:
update readme.md

yuerenwa committed:
update readme.md

hytao committed:
update description for UrlRewrite Action (#7715)

* update comment

* update

yuerenwa committed:
modify readme.md

yuerenwa committed:
modify readme.md

yuerenwa committed:
fix avacado

yuerenwa committed:
fix avacado

yuerenwa committed:
fix avacado 2

yuerenwa committed:
fix avacado 2

kingsleyAzure commented:
How Can I fix this prettier check thing ? I tried tutorials here and not working:
https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/ci-fix.md

leonardbf committed:
New swagger for 2019-08-01 API (#7660)

SanjaMalesevic committed:
AdministratorName parameter should take value which is defined in enum  (#7377)

* AdministratorName parameter should take constant value defined in enum

* Code review fixes

* Code review fixes

* Fix tabs and spaces

* Fix example with get

* revert example with get

* Fix examples

majastrz commented on specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/examples/ListManagedInstanceOperations.json:
```diff
@@ -0,0 +1,72 @@
+            }
+          },
+          {
+            "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/sqlcrudtest-7398/providers/Microsoft.Sql/managedInstances/sqlcrudtest-4645/operations/22222222-2222-2222-2222-222222222222",
```
>/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/sqlcrudtest-7398/providers/Microsoft.Sql/managedInstances/sqlcrudtest-4645/operations/22222222-2222-2222-2222-222222222222 [](start = 19, length = 189)

You seem to be returning IDs in the list call, but there's no declared path that lets you GET a single operation by that ID.

kingsleyAzure replied:
Yes. In ManagedInstanceOperations (and ServerOperations). By calling this API. the operations on server and MI will be returned. Fetching operation status by ID is currently not supported.

majastrz commented on specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/databases.json:
```diff
@@ -0,0 +1,1190 @@
+{
+  "swagger": "2.0",
```
>swagger [](start = 3, length = 7)

Next time, please make the unmodified version of the previous API version swagger as the first commit. It makes diffing the delta much easier. (No action needed for this PR.)

kingsleyAzure replied:
ok

majastrz commented on specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/databases.json:
```diff
@@ -0,0 +1,1190 @@
+            "description": "Successfully upgraded the data warehouse."
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 InvalidCollation - Collation is not recognized by the server.\n\n * 400 ProvisioningDisabled - Displays error message from resources operation authorizer as is, without changes\n\n * 400 UnableToResolveRemoteServer - The remote partner server name could not be resolved due to an invalid server name or DNS connectivity issues.\n\n * 40 ...
```
>/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/inaccessibleDatabases [](start = 5, length = 133)

You should also have a path that lets you query a single inaccessible database by ID.

majastrz commented on specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/databases.json:
```diff
@@ -0,0 +1,1190 @@
+            "description": "Successfully upgraded the data warehouse."
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 InvalidCollation - Collation is not recognized by the server.\n\n * 400 ProvisioningDisabled - Displays error message from resources operation authorizer as is, without changes\n\n * 400 UnableToResolveRemoteServer - The remote partner server name could not be resolved due to an invalid server name or DNS connectivity issues.\n\n * 40 ...
```
>GRS [](start = 13, length = 3)

What about RAGRS? (Read-Access Geo-Redundant Storage). Would you support that also?

kingsleyAzure replied:
I did not own RAGRS. I created 4 new APIs under 2019-06-01-preview and generate the swagger file

majastrz commented on specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/managedDatabases.json:
```diff
@@ -0,0 +1,717 @@
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
```
>paths [](start = 3, length = 5)

I noticed that you removed GET /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/managedInstances/{managedInstanceName}/databases/{databaseName}/restoreDetails/{restoreDetailsName} when comparing with 2018-06-01-preview version. Is that intentional?

kingsleyAzure replied:
I did not do any manual modification on this. I ran generate swagger so that the manifest gets generated. Should I manually add it back?

majastrz replied:
I wouldn't know. Will those be routes that you support in this API version? If yes, then you probably should investigate why your generation logic excluded it. 

---
In reply to: [343892400](https://github.com/Azure/azure-rest-api-specs/pull/7551#discussion_r343892400) [](ancestors = 343892400)

majastrz commented on specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/serverOperations.json:
```diff
@@ -0,0 +1,315 @@
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/operations": {
```
>/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/operations [](start = 5, length = 122)

How do you list a single operation?

kingsleyAzure replied:
This API will list only particular operations, we currently do not support list a single operations using this api. when portal calls this API. it will return only the make all database accessible operations. but we can expand the supported operation list to return by this API.

majastrz reviewed (changes requested):
I added a couple of comments. Please take a look.

majastrz reviewed (approved):
I'm fine with the changes as is from the ARM side, but you need to follow up why the route got removed in case it wasn't intentional as it can lead to unintended breakage in the SDK.

yuerenwa committed:
Merge branch 'TDEBYOKapiswaggers' of https://github.com/kingsleyAzure/azure-rest-api-specs into TDEBYOKapiswaggers

yuerenwa committed:
fix prettier
